kedro/framework/cli/starters.py

```diff
@@ -93,6 +93,24 @@ class KedroStarterSpec:  # noqa: too-few-public-methods
     "An optional directory inside the repository where the starter resides."
 )
 
+# TODO; Insert actual link to the documentation.
+ADDON_ARG_HELP = """
+Select which add-ons you'd like to include. By default, none are included.
+To read more about these add-ons and what they do visit: kedro.org/{insert-documentation}
```

Comment:
```suggestion
To found out more about these add-ons visit: kedro.org/{insert-documentation}
```

Reply:
Is it possible to have it like this? It is better because it's got the action first that the reader needs to take.